### PR TITLE
chore(sdk-core): add clarity to other commonKeychain util

### PR DESCRIPTION
**Pull request by kxl4126**
Ticket: BG-0000

### CODE REVIEW

File: modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts [hunk 1]
```diff
@@ -1183,7 +1183,12 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       .result();
   }
 
-  static publicKeyFromCommonKeychain(commonKeychain: string) {
+  /**
+   * util function that checks that a commonKeychain is valid and can ultimately resolve to a valid public key
+   * @param commonKeychain - a user uploaded commonKeychain string
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts:1188]: Does this function throw any errors?

Comment (zahin-mohammad) [reply modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts:1188]: Can you also please add non-happy path tests?

**Review by zahin-mohammad — CHANGES_REQUESTED** [review]

File: modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts [hunk 2]
```diff
@@ -1183,7 +1183,13 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
...
+   * @throws if the commonKeychain is invalid length or invalid format
+   */
+
+  static getUncompressedPublicKeyFromCommonKeychain(commonKeychain: string) {
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts:1192]: This naming is little clearer
```suggestion
  static validateCommonKeychainPublicKey(commonKeychain: string) {
```

**Review by zahin-mohammad — APPROVED** [review]

**Review by alebusse — COMMENTED** [review]
I dont think you need `chai`, you could use `should` in a similar way

**Review by alebusse — APPROVED** [review]

**Pull request merged**